### ISSUE TITLE
How to set the starting directory?

### ISSUE THREAD

**Issue by AtmanActive** [boyter/cs#21]
Hi,

Thank you for developing this great tool.

Reading through the documentation, I can see that the recommended way to use the tool is to run it in the working directory. But that implies that I need to add cs.exe to each of my projects just to be able to search the code. That feels unacceptable to me. Hence, I was wondering, how can I use this as a tool, meaning, how can I keep cs.exe in some directory A, but still order it to scan the source code in some other directory B.

This [closed issue](https://github.com/boyter/cs/issues/12) mentions this functionality but it doesn't work for me. When I issue a full path to source code project as a first parameter, cs.exe just exists like nothing happened.

My use case is to be able to create several shortcuts (or bat files, or shell files or whatever) where path to cs.exe would always be the same one, but the path to source code project would be different for each project, sometimes even on different drives.

 Thanks.

**Comment by AtmanActive** [comment]
Was testing v1.1.1

**Comment by boyter** [comment]
Hi @AtmanActive this was one of those things I dropped so I could finally push out a version 1 release.

I have added it into the mainline if you want to build from master. Otherwise I will release it as a 1.2.0 release in a few hours (as soon as I have stable internet again).

It will work like the below, were I am searching over redis from the cs development directory.

```
$ pwd
/Users/boyter/Documents/projects/cs
$ cs --dir /Users/boyter/Documents/projects/redis FINDTHIS
```

This will work in console mode, via tui or http.
